app/javascript/packages/components/troubleshooting-options.tsx

```diff
@@ -1,12 +1,14 @@
 import type { ReactNode } from 'react';
-import { BlockLink } from '@18f/identity-components';
+import { BlockLink, BlockSubmitButton } from '@18f/identity-components';
 import { useI18n } from '@18f/identity-react-i18n';
 
 export interface TroubleshootingOption {
   url: string;
 
   text: ReactNode;
 
+  isContinue?: boolean;
```

Comment:
It means that the option will be functionally identical to a `<FormStepsButton.Continue />` component rather than an external link or an internal link to a page with specific address.

Reply:
I removed `isContinue` and am now relying on the presnece of `url` to decide whether to render a Link or a button in a71df2bb36d952c69a2be78493b56c0b72d736ba